crates/ruma-federation-api/src/transactions/edu.rs

```diff
@@ -258,7 +258,7 @@ pub struct DirectDeviceContent {
     /// The contents of the messages to be sent. These are arranged in a map
     /// of user IDs to a map of device IDs to message bodies. The device ID may
     /// also be *, meaning all known devices for the user.
-    pub messages: BTreeMap<UserId, BTreeMap<DeviceIdBox, RoomKeyToDeviceEventContent>>,
+    pub messages: BTreeMap<UserId, BTreeMap<DeviceIdOrAllDevices, Box<RawJsonValue>>>,
```

Comment:
Why `Box<RawJsonValue>` instead of `Raw<Something>`?

Reply:
I did this to match ruma-client-api/src/r0/to_device/send_event_to_device.rs

Reply:
Alright, I might update both at the same time then.